### DISCUSSION TITLE
API and Binary

### DISCUSSION THREAD

**Discussion by heades** [cvc5/cvc5#8880] [Q&A] [answered]
hi, I was wondering if the binary is necessary for the API to function?

**Comment by 4tXJ7f** [comment, marked as answer]
Hi @heades, no, the binary is not necessary for using the API. The only files that you should need are the header files, libcvc5, and the library files for dependencies (GMP and LibPoly). Please let us know if you run into any issues with that.

**Reply by heades** [reply]
Thanks! Exactly what I was needing to know.
